File: bench/prime/prime.js
function is_prime(n) {
    if (n <= 1) { return false }
    for (let i = 2; i < n; i++) {
        if (n % i == 0) { return false }
    }
    return true;
}

function count_primes(limit) {
    let count = 0;
    for (let i = 1; i < limit; i++) {
        if (is_prime(i)) {
            count = count + 1;
        }
    }
    return count;
}

console.log(count_primes(10000))

File: bench/prime/prime.py
def is_prime(n):
    if n <= 1: return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0: return False
    return True

def count_primes(limit):
    count = 0
    for i in range(1, limit):
        if is_prime(i):
            count += 1
    return count

print(count_primes(10000))